oc-cluster

```diff
@@ -253,7 +303,15 @@ function up {
 
     echo "$CMDLINE" > $OPENSHIFT_PROFILES_DIR/$_profile/run
     echo "$(<$OPENSHIFT_PROFILES_DIR/$_profile/run)"
-    $CMDLINE
+    if [ ! -z $DEV_DOCKER_HOST ]; then
+      echo "Running in developer mode, using DOCKER_HOST=$DEV_DOCKER_HOST"
+      if [[ "$__PLATFORM" == "linux" ]]; then
+        CMDLINE="DOCKER_HOST=$DEV_DOCKER_HOST $CMDLINE"
+      fi
+      $CMDLINE -e DOCKER_HOST=$DEV_DOCKER_HOST
```

Comment:
Wouldn't it be better to include different mount templates and use the one that's appropriate for your use case?